fem/fe.cpp

```diff
@@ -1750,6 +1750,242 @@ void BiQuad2DFiniteElement::ProjectDelta(int vertex, Vector &dofs) const
 #endif
 }
 
+
+H1Ser_QuadrilateralElement::H1Ser_QuadrilateralElement(const int p)
+   : ScalarFiniteElement(2, Geometry::SQUARE, (p*p + 3*p +6) / 2, p,
+                         FunctionSpace::Qk)
+{
+   // ***  Do we want FunctionSpace = Pk or Qk for serendipity elements?   ***
+
```

Comment:
None of them is technically correct, but the natural choice is Qk for quads/hexes and Pk for simplices

Reply:
One option could be to add a new value for serendipity spaces, we can call it e.g. `Sk`.